resources/js/i18n/en.json

```diff
@@ -761,16 +769,40 @@
         },
         "importer": {
             "title": "Data-Importer",
+            "entity_settings": "Entity Settings",
+            "attribute_mapping": "Attribute Mapping",
             "upload_csv_info": "Select a CSV file to import data.",
             "drop_csv_file": "Drop CSV file here",
+            "csv_file_error": "CSV-Datei konnte nicht gelesen werden.",
             "selected_entity_type": "Entity-Type",
             "column_entity_name": "Entity Name-Column",
             "column_entity_parent": "Entity Parent-Column",
             "import_btn": "Start Import",
+            "validate": "Validate",
             "no_missing_values": "No missing values for this attribute. You are all set!",
             "missing_required_values": "{miss} out of 1 entry is empty! This option does not allow empty values. Please fix this. | {miss} out of {total} entries are empty! This option does not allow empty values. Please fix this.",
             "missing_non_required_values": "{miss} out of 1 entry is empty! Filled values are not required for this attribute, so you can go on but may want to re-check. | {miss} out of {total} entries are empty! Filled values are not required for this attribute, so you can go on but may want to re-check.",
-            "success": "Import of 1 entry finished | Import of {cnt} entries finished"
+            "success": "Import of 1 entry finished | Import of {cnt} entries finished",
+            "validation": {
+                "title": "Validation",
+                "mixed": "Some entities do already exist and will be updated, the remaining will be created!",
+                "create": "All items will be created as new entities.",
+                "update": "All entites do already exist and will be updated.",
+                "multiple_error": "Some values match multiple entities and can not be imported.",
+                "no_items": "There are no items in your import"
+            },
+            "indicator": {
+                "missing_value": "No value set",
+                "ok": "This value is set correctly"
+            },
```

Comment:
same as in german; missing info for the user

Reply:
Fixed in e1f017ea68742e1c0d73bf36848c7e753edfb3c9